runtime/jcl/common/shared.c

```diff
@@ -255,15 +255,12 @@ createCPEntries(JNIEnv* env, jint helperID, jint urlCount, J9ClassPathEntry*** c
 		UDATA cpeType = 0;
 		char* correctPath = NULL;
 
-		cpeType = getCpeTypeForProtocol((char*)urlArrayElements[i].protocolChars, urlArrayElements[i].protocolLen, urlArrayElements[i].pathChars, urlArrayElements[i].pathLen);
+		cpeType = getCpeTypeForProtocol(env, (char*)urlArrayElements[i].protocolChars, urlArrayElements[i].protocolLen,
+			urlArrayElements[i].pathChars, urlArrayElements[i].pathLen, &correctPath);
```

Comment:
When splitting, each argument gets a line by itself (indented two tabs more than the starting line); reversing the local declarations will allow `cpeType` to be initialized with the right value, the signature change to `getCpeTypeForProtocol()` eliminates the need to cast the second argument:
```c
		UDATA cpeType = getCpeTypeForProtocol(
				env,
				urlArrayElements[i].protocolChars,
				urlArrayElements[i].protocolLen,
				urlArrayElements[i].pathChars,
				urlArrayElements[i].pathLen,
				&correctPath);
```